JitsiConference.js

```diff
@@ -245,7 +245,32 @@ JitsiConference.prototype._init = function(options = {}) {
 
     this.room.updateDeviceAvailability(RTC.getDeviceAvailability());
 
-    this.e2eping = new E2ePing(this, config);
+    this.e2eping = new E2ePing(
```

Comment:
i think this looks better, thanks.  only tweak i'd say is move this code to a helper method.